containers/flatpak/test/test-browser

```diff
@@ -83,5 +98,18 @@ class BrowserTest(unittest.TestCase):
         self.assertEqual(self.run_js("test-browser-login.js"), "page-load")
         self.assertEqual(self.run_js("test-browser-overview.js"), "PASS")
 
+    def testRecentHosts(self):
+        self.assertIn(self.run_js("test-browser-wait-init.js"), ["found", "page-load"])
+
+        # try to log in by clicking on `localhost`
+        self.assertEqual(self.run_js("test-browser-login-recent.js"), "page-load")
```

Comment:
You forgot to git add the new test files here, so I can't reproduce this locally. This is probably also why the test fails on GitHub:

> FileNotFoundError: [Errno 2] No such file or directory: '/var/home/martin/upstream/cockpit/containers/flatpak/test/test-browser-login-recent.js'